xbmc/addons/AddonManager.cpp

```diff
@@ -320,6 +320,24 @@ bool CAddonMgr::Init()
 
   FindAddons();
 
+  // disable some system addons by default because they are optional
+  VECADDONS addons;
+  GetAddons(ADDON_PVRDLL, addons);
+  GetAddons(ADDON_AUDIODECODER, addons);
+  GetAddons(ADDON_AUDIOENCODER, addons);
+  std::string systemAddonsPath = CSpecialProtocol::TranslatePath("special://xbmc/addons");
+  for (size_t i = 0; i<addons.size(); i++)
+  {
```

Comment:
@xhaggi I agree to new coding styles but I am not fully accustomed to them. In this stage I try to fix bugs not introducing new ones because of learning curve.